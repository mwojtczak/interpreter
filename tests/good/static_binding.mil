function main () {
	i = 20;
	dec = function (var x){
		x = x - 1;
		print(i);
	}
	[local i];
	{
		i = 5;
		while (i > 0) {
			dec(i);
			print(i);
		}
	}
}