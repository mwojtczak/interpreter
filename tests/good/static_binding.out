20
4
20
3
20
2
20
1
20
0
